examples/data-objects/monaco/webpack.config.cjs

```diff
@@ -4,107 +4,68 @@
  */
 
 const fluidRoute = require("@fluid-example/webpack-fluid-loader");
-const path = require("path");
 const { merge } = require("webpack-merge");
 const MonacoWebpackPlugin = require("monaco-editor-webpack-plugin");
 
 module.exports = (env) => {
-	const isProduction = env?.production;
-	return merge(
-		{
-			entry: {
-				main: "./src/index.ts",
-			},
-			mode: "production",
-			devtool: "source-map",
-			module: {
-				rules: [
-					{
-						test: /\.tsx?$/,
-						use: [
-							{
-								loader: "ts-loader",
-								options: {
-									compilerOptions: {
-										module: "esnext",
```

Comment:
This config caused ` TS5110: Option 'module' must be set to 'Node16' when option 'moduleResolution' is set to 'Node16'.` So I made it mased on our default config. The dependencies still cause source maps issues which make the app not work when manually testing it), so I had to keep source map suppression.